tests/test_assemblies.py

```diff
@@ -49,14 +41,28 @@ def test_int_multi_coord(self):
     @pytest.mark.skip(reason="not implemented")
     def test_multi_dim(self):
         d = DataAssembly([[1, 2, 3], [4, 5, 6], [7, 8, 9], [10, 11, 12]],
-                         coords={'a': ['a', 'a', 'b', 'b'],
+                         coords={'a': ['alpha', 'alpha', 'beta', 'beta'],
```

Comment:
if these change, the ones below (`assert g.equals(...)`) should also change no? (this test is skipped)

Reply:
let's just delete this test

Reply:
Reverted.  I tried making a function to produce DataArrays with more typical coordinates.  I'll remove it if it's gross.